### DISCUSSION TITLE
Donde se suben las Practicas de Front End

### DISCUSSION THREAD

**Discussion by 79155637** [Launch-X-Latam/MisionFrontEnd#451] [Team Posts]
Buenas noches señores sherpa digital: primero agradecerles por las clases de Frontend de Rodrigo Martinez, pero he visto todos los videos y estoy haciendo la practica del Abogabot correspondiente a la introducción y siempre Rodrigo dijo esperáramos a que el diera las instrucciones, y de repente dice que la subamos a  GitHub.  La pregunta es: bajo que parámetros lo sub, a mi GitHub o el de LaunchX (no creería lo permita)?? o tienen algún instructivo en su repositorio de FrontEnd de Launch X (pero me lo he perdido), por favor me pueden colaborar.

**Comment by Sebastian-Longhitano** [comment]
De momento te conviene crearte tu propio repositorio e ir subiendo todo al mismo!
Saludos,

**Comment by mafer23** [comment]
has un pull request al repositorio

**Comment by gumodi35** [comment]
Bueno yo no soy quien 🧑‍🏫  para decirte que no, pero un PR 🎏 no creo que sea exactamente lo que se pide, te sugiero hacer un repositorio en tu GitHub o en su defecto hacer un Fork ©️ del repo y hay ya subir tus prácticas. 🐰 Así se hizo anteriormente. Saludos!!

**Comment by mafer23** [comment]
![pullrequest](https://user-images.githubusercontent.com/39041288/196015020-82bb2062-2d6b-4760-acaf-9f35fd24be4a.png)

**Comment by 79155637** [comment]
Gracias, ya lo hice como me lo sugieres, quedo pendiente  de si aprueban mis cambios

**Comment by MarcoAntonio159** [comment]
Hola @mafer23, entiendo que nos has dado el paso, para poder subir nuestras actividades, eh buscado informacion, para realizar todos los pasos indicados, pero quisiera saber si existe algun instructivo donde pueda seguir los pasos, para personas como yo, que nunca habian usado GitHub, ya que es una herramienta nueva.

Con muchos videos, y podido encontrar que  como realizar un fork, y como encontrarlo en mi repositorio, pero cuando indicas descargar el repositorio, te refieres a descargarlo fisicamente a mi equipo?

Al crear una nueva rama, este lo realizo dentro dentro del fork que eh realizado?
![imagen](https://user-images.githubusercontent.com/19269827/196702405-6383c2b1-e32a-4127-9d84-8a8a96369ece.png)

Entiendo que los cambios que se realizan, se hace desde la nueva rama? pero tengo que crear nuevas carpetas o dentro de la seccion de practicas, puedo editarlo y ahi subir mi tarea?

Cuando te refieres a PR, es un pull request?  o Push request?

Como actualizo los cambios en mi repositorio local?

A que te refieres con que subes esas actualizaciones al remoto?

Son terminos que no me quedan muy claros, y quisiera saber si podrias ayudarme con ello, o si existe manera de que te pueda contactar para que me brindes la ayuda correspondiente.

De antemano, muchas gracias.

**Comment by jjmillar** [comment]
> Hola @mafer23, entiendo que nos has dado el paso, para poder subir nuestras actividades, eh buscado informacion, para realizar todos los pasos indicados, pero quisiera saber si existe algun instructivo donde pueda seguir los pasos, para personas como yo, que nunca habian usado GitHub, ya que es una herramienta nueva.
> 
> Con muchos videos, y podido encontrar que como realizar un fork, y como encontrarlo en mi repositorio, pero cuando indicas descargar el repositorio, te refieres a descargarlo fisicamente a mi equipo?
> 
> Al crear una nueva rama, este lo realizo dentro dentro del fork que eh realizado?
> 
> 
> Entiendo que los cambios que se realizan, se hace desde la nueva rama? pero tengo que crear nuevas carpetas o dentro de la seccion de practicas, puedo editarlo y ahi subir mi tarea?
> 
> Cuando te refieres a PR, es un pull request? o Push request?
> 
> Como actualizo los cambios en mi repositorio local?
> 
> A que te refieres con que subes esas actualizaciones al remoto?
> 
> Son terminos que no me quedan muy claros, y quisiera saber si podrias ayudarme con ello, o si existe manera de que te pueda contactar para que me brindes la ayuda correspondiente.
> 
> De antemano, muchas gracias.

Hola estimada! Te dejo este link de un muy buen video sobre GIT https://www.youtube.com/watch?v=VdGzPZ31ts8 . Al final la tecnologia que se usa es Git, y github es un lugar donde alojar todo el tema del git. Te recomiendo ponerle mucha atencion al min 24, donde hablan de el flujo de trabajo de Git, para que te des cuenta de que hacer el FORK, como menciona mafer, NO es lo que hay que hacer. Lo que tienes que hacer es simplemente crear un repositorio tuyo nuevo y subirlos ahí. En el OnBoarding estan unas secciones de Github y como usarlo https://github.com/Launch-X-Latam/OnBoarding . Volviendo atrás, no hagas lo del fork que no tiene sentido y no vas a llegar a niun lado.

**Comment by OscarRufino** [comment]
@InerGiveUp Tambien te recomiendo darle una vuelta a los primeros capítulos los del libro de Git(Documentación), si entiendes lo que es Git y los fundamentos de cómo funciona, seguramente te será mucho más fácil usar Git y Github.

En la página oficial de Git puedes ver y descargar la version en español: https://git-scm.com/book/es/v2

**Comment by MarcoAntonio159** [comment]
> Hola @mafer23, entiendo que nos has dado el paso, para poder subir nuestras actividades, eh buscado informacion, para realizar todos los pasos indicados, pero quisiera saber si existe algun instructivo donde pueda seguir los pasos, para personas como yo, que nunca habian usado GitHub, ya que es una herramienta nueva.
> 
>     Con muchos videos, y podido encontrar que como realizar un fork, y como encontrarlo en mi repositorio, pero cuando indicas descargar el repositorio, te refieres a descargarlo fisicamente a mi equipo?
> 
>     Al crear una nueva rama, este lo realizo dentro dentro del fork que eh realizado?
> 
>     Entiendo que los cambios que se realizan, se hace desde la nueva rama? pero tengo que crear nuevas carpetas o dentro de la seccion de practicas, puedo editarlo y ahi subir mi tarea?
> 
>     Cuando te refieres a PR, es un pull request? o Push request?
> 
>     Como actualizo los cambios en mi repositorio local?
> 
>     A que te refieres con que subes esas actualizaciones al remoto?
> 
>     Son terminos que no me quedan muy claros, y quisiera saber si podrias ayudarme con ello, o si existe manera de que te pueda contactar para que me brindes la ayuda correspondiente.
> 
>     De antemano, muchas gracias.
> 
> Hola estimada! Te dejo este link de un muy buen video sobre GIT https://www.youtube.com/watch?v=VdGzPZ31ts8 . Al final la tecnologia que se usa es Git, y github es un lugar donde alojar todo el tema del git. Te recomiendo ponerle mucha atencion al min 24, donde hablan de el flujo de trabajo de Git, para que te des cuenta de que hacer el FORK, como menciona mafer, NO es lo que hay que hacer. Lo que tienes que hacer es simplemente crear un repositorio tuyo nuevo y subirlos ahí. En el OnBoarding estan unas secciones de Github y como usarlo https://github.com/Launch-X-Latam/OnBoarding . Volviendo atrás, no hagas lo del fork que no tiene sentido y no vas a llegar a niun lado.

Muchas gracias Julian,

Voy a revisar el vidoe y la documentacion brindada, y con ello, creare mi propio repositorio, soloe spero al final del curso, nos brinden una pagina para copiar el enlace de nuestros repositorios, como vi que lo indicaron algunos compañeros.

Saludos.

**Comment by MarcoAntonio159** [comment]
> @InerGiveUp Tambien te recomiendo darle una vuelta a los primeros capítulos  del libro de Git(Documentación), si entiendes lo que es Git y los fundamentos de cómo funciona, seguramente te será mucho más fácil usar Git y Github.
> 
> En la página oficial de Git puedes ver y descargar la version en español: https://git-scm.com/book/es/v2

Hola Oscar,

revisare la documentacion.

Saludos.
Gracias.!
